src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -415,6 +415,7 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
 
   // Store getter
   @Getter getOrgPeople!: OrgPersonIF[]
+  @Getter getResource!: ResourceIF
```

Comment:
1. I don't see this type being imported...?

2. Using a resource here, while most other conditions are passed in as props, is mixing architectures. It would be better to have just 1 architecture in this component. I suggest passing in a prop for the condition you need, and in the parent you can look at the resource file flag and pass in the prop.

Reply:
I want to see what you've changed here.

Reply:
Give me a few minutes to review your latest code. I'll update this conversation accordingly.

Reply:
OK, this component no longer looks at the resource file. Now it uses props for the new logic. 👍